src/UI/walletConnect/WalletConnectLoginButton/index.tsx

```diff
@@ -32,34 +36,43 @@ export const WalletConnectLoginButton = ({
   wrapContentInsideModal = true,
   redirectAfterLogin = false,
   buttonClassName,
-  className = 'wallect-connect-login',
+  className = styles.wallectConnectLogin,
   lead = 'Scan the QR code using Maiar',
   token,
   hideButtonWhenModalOpens = false
 }: WalletConnectLoginButtonPropsType) => {
-  const [showLoginModal, setShowLoginModal] = useState(false);
+  const [canShowLoginModal, setCanShowLoginModal] = useState(false);
+  const { handleShowModal, handleHideModal } = useDappModal();
+
   const generatedClasses = getGeneratedClasses(
     className,
     shouldRenderDefaultCss,
     {
-      wrapper: `btn btn-primary px-sm-4 m-1 mx-sm-3 ${
-        buttonClassName != null ? buttonClassName : ''
-      }`,
-      loginText: 'text-left'
+      wrapper: `${classNames(
+        globalStyles.btn,
+        globalStyles.btnPrimary,
+        globalStyles.px4,
+        globalStyles.m1,
+        globalStyles.mx3,
+        buttonClassName != null && buttonClassName
```

Comment:
maybe reuse wrapper style since it was defined above also

Reply:
Please see my answer [here](https://github.com/ElrondNetwork/dapp-core/pull/275#discussion_r904600895)